ui/components/src/TimeSeriesTooltip/TimeSeriesTooltip.tsx

```diff
@@ -92,7 +113,31 @@ export const TimeSeriesTooltip = React.memo(function TimeSeriesTooltip({
           transform: cursorTransform,
         }}
       >
-        <TooltipContent focusedSeries={focusedSeries} wrapLabels={wrapLabels} />
+        <TooltipContent
+          focusedSeries={focusedSeries}
```

Comment:
I might be wrong about this, but I'm thinking we could just call this prop `series` and pass in the `series={focusedSeries}`

If I'm getting it right, from `TooltipContent`'s perspective, it's receiving the list of series to render, and it will render whatever it gets.

Here in the parent, we are choosing to only pass in the `focusedSeries`.